scenarios/kubernetes_e2e_test.py

```diff
@@ -109,7 +109,8 @@ def test_name_filled(self):
         """Return the cluster name if set."""
         name = 'foo'
         build = '1984'
-        actual = kubernetes_e2e.cluster_name(name, build)
+        os.environ['BUILD_NUMBER'] = build
```

Comment:
> Note: Use of $BUILD_NUMBER is deprecated. Please use $BUILD_ID instead.

https://github.com/kubernetes/test-infra/tree/master/prow#job-environment-variables
I do not know *why* one is preferred over the other, but from the env docs we should prefer BUILD_ID here.

Reply:
I think either would work, but BUILD_ID should be used. BUILD_NUMBER will be removed at some point.